app/proto/auth.proto

```diff
@@ -243,3 +247,12 @@ message ContributorForm {
   repeated string contribute_ways = 5;
   string expertise = 6;
 }
+
+message UnsubscribeReq {
+  bytes payload = 1;
+  bytes sig = 2;
+}
+
+message UnsubscribeRes {
+  string response = 1;
```

Comment:
Please describe that actually getting an UnsubscribeRes always means that the operation was successful, regardless of the string.